client/client.go

```diff
@@ -305,10 +299,13 @@ func (c *Client) downloadTimestamp() error {
 func (c *Client) downloadSnapshot() error {
```

Comment:
Why would we have a specific `downloadSnapshot()` when we have a `downloadSigned()` ?